packages/core/src/components/toast/toast.scss

```diff
@@ -102,10 +103,6 @@
       color: var(--tds-toast-subheadline-color);
       font: var(--tds-detail-02) !important;
       letter-spacing: var(--tds-detail-02-ls) !important;
-      max-width: 252px;
-      white-space: nowrap;
-      overflow: hidden;
-      text-overflow: ellipsis;
```

Comment:
Might be good to highlight this in the announcement since this could be considered breaking. People might have a lot of content in their toasts which they expect to be truncated.